receiver/hostmetricsreceiver/internal/scraper/cpuscraper/metadata.yaml

```
@@ -27,7 +27,7 @@ metrics:
 
   system.cpu.utilization:
     enabled: false
-    description: Difference in system.cpu.time since the last measurement, divided by the elapsed time and number of logical CPUs.
+    description: Difference in system.cpu.time since the last measurement per logical CPU, divided by the elapsed time.
```

could also help w/ clarity

```suggestion
    description: Difference in system.cpu.time since the last measurement per logical CPU, divided by the elapsed time (value in interval [0,1]).
```